app/main.py

```diff
@@ -1,3 +1,47 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f'Distance: {self.km} kilometers.'
+
+    def __repr__(self):
+        return f'Distance(km={self.km})'
+
+    def __add__(self, other):
+        return Distance(self.km + other.km) \
+            if isinstance(other, Distance) else Distance(self.km + other)
+
+    def __iadd__(self, other):
+        self.km += other.km \
+            if isinstance(other, Distance) else other
+        return self
+
+    def __mul__(self, other):
+        return Distance(self.km * other)
+
+    def __truediv__(self, other):
+        return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other):
+        return self.km < other.km \
+            if isinstance(other, Distance) else self.km < other
+
+    def __gt__(self, other):
+        return self.km > other.km \
+            if isinstance(other, Distance) else self.km > other
+
+    def __eq__(self, other):
+        return self.km == other.km \
+            if isinstance(other, Distance) else self.km == other
+
+    def __le__(self, other):
+        return self.km <= other.km \
```

Comment:
You may refactor your code in this way:
return not self > other
*because it's a magic method)